paymentsheet/src/main/java/com/stripe/android/link/account/LinkAccountUtil.kt

```diff
@@ -4,3 +4,12 @@ import com.stripe.android.link.LinkAccountUpdate
 
 internal val LinkAccountManager.linkAccountUpdate: LinkAccountUpdate
     get() = LinkAccountUpdate.Value(linkAccount.value)
+
+internal fun LinkAccountUpdate.updateLinkAccount(linkAccountHolder: LinkAccountHolder) {
+    when (this) {
+        is LinkAccountUpdate.Value -> {
+            linkAccountHolder.set(linkAccount)
+        }
+        LinkAccountUpdate.None -> Unit
+    }
+}
```

Comment:
extracted this from `LinkConfirmationDefinition` as it comes handy on many places now.